arex-storage-web-api/src/main/java/com/arextest/storage/metric/AgentWorkingMetricService.java

```diff
@@ -39,14 +39,18 @@ public class AgentWorkingMetricService {
   private final AgentWorkingService agentWorkingService;
   private final MockSourceEditionService editableService;
   private final InvalidReplayCaseService invalidReplayCaseService;
+  private final ScheduledExecutorService invalidCaseDelayedPool;
 
   public AgentWorkingMetricService(AgentWorkingService agentWorkingService,
                                    MockSourceEditionService editableService,
-      List<MetricListener> metricListeners, InvalidReplayCaseService invalidReplayCaseService) {
+                                   List<MetricListener> metricListeners,
+                                   InvalidReplayCaseService invalidReplayCaseService,
+                                   ScheduledExecutorService invalidCaseDelayedPool) {
```

Comment:
`invalidReplayCaseService/invalidCaseDelayedPool` should not be in `MetricClass`